examples/test-cases/error-test/error_test.py

```diff
@@ -48,8 +48,7 @@ def get_successors(self, state):
         export=["DOWNWARD_REPO", "DOWNWARD_BENCHMARKS"],
         extra_options="#SBATCH --cpus-per-task=2")
 
-evaluator_filename = os.path.join(os.path.dirname(get_script_path()),
-                                  "evaluator.py")
+evaluator_filename = get_script_path().parent / "evaluator.py"
```

Comment:
The variable should be called `evaluator_path` or just `evaluator` now. (Probably also in all other test cases and in the documentation, I'll not comment on each one individually)